pkgs/by-name/ap/application-title-bar/package.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, kdePackages
+}:
+
+stdenv.mkDerivation rec {
+  pname = "application-title-bar";
+  version = "0.6.2";
+
+  src = fetchFromGitHub {
+    owner = "antroids";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-KKODCfLL+T4NdflxUFP++TO3OtQWJpLHAnUVqmvIzrg=";
+  };
+
+  buildInputs = with kdePackages; [ qttools kconfig ];
+
+  dontWrapQtApps = true;
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/{bin,share/plasma/plasmoids/com.github.antroids.application-title-bar}
+    cp -r $src/package/* $out/share/plasma/plasmoids/com.github.antroids.application-title-bar
+    ln -s ${kdePackages.kconfig}/bin/{kwriteconfig6,kreadconfig6} $out/bin
+    ln -s ${kdePackages.qttools}/bin/qdbus $out/bin
```

Comment:
instead of symlinking these to /bin/, which might interfere with a FHS user env, have you tried adding these packages to `propagatedUserEnvPkgs` or `propagatedBuildInputs`?

Reply:
qttools should also be part of plasma already

Reply:
> instead of symlinking these to /bin/, which might interfere with a FHS user env, have you tried adding these packages to `propagatedUserEnvPkgs` or `propagatedBuildInputs`?

I've tried propagatedBuildInputs, but it did not work back when I tried, but I tried propagatedUserEnvPkgs now, and it works.